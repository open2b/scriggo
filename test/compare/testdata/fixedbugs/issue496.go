// paniccheck

package main

func main() {
	var x uint8 = 4
	var y = [1]uint32{1}
	print(y[128+x])
}