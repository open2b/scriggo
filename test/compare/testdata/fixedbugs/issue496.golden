panic: runtime error: index out of range [132] with length 1